metricbeat/module/kubernetes/_meta/remote-debugger/README.md

```diff
@@ -0,0 +1,125 @@
+# README
+
+This readme explain how to remote debug metricbeat running on docker/kubernetes from your laptop with VisualStudioCode
+
+A common requirement for both remote debugging in docker or kubernetes is to have a file `.vscode/launch.json` on your local machine. 
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Connect to server",
+            "type": "go",
+            "request": "attach",
+            "mode": "remote",
+            "debugAdapter": "dlv-dap",
+            "port": 56268,
+            "host": "127.0.0.1",
+            "showLog": true,
+            "trace": "trace",
+            "cwd": "${workspaceFolder}",
+            "substitutePath": [
+                { 
+	                "from": "${workspaceFolder}",
+					# NOTE: REPLACE `<absolute path>` with the absolute path of the root folder
```

Comment:
JSON does not support comments, so this would not be a valid JSON snippet. Can we move this comment to before or after the JSON block?

Reply:
sure. thanks for noticing